src/pages/blog/blogArchives/blogArchives.scss

```diff
@@ -1,8 +1,16 @@
 #blog-archive-container {
-  display: flex;
-  flex-wrap: wrap;
   padding: 5em 3em;
   margin: 0 auto;
+
+  #blog-archive-header {
+    font-size: 3em;
+    padding-bottom: 2rem;
+  }
+  #blog-archives{
+    display: flex;
+    flex-direction: row;
```

Comment:
Flex direction row is the default for `display: flex;` so it isn't needed here. However, we do need to re-implement `flex-wrap: wrap;` which was removed from `#blog-archive-container`. Without wrapping, the tiles do not fit correctly on tablet and mobile formats.